Yoast/javascript
# 457 determine unique stem

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds the master function for Dutch stemming.

## Relevant technical choices

*  It calls other smaller functions such as smaller stemmer function, participle stemmer, words ending in t/d stemmer.

## Test instructions

This PR can be tested by following these steps:

* Add more words to the specs
* Run `yarn test` and make sure there is no failing test

## Quality assurance


Fixes #457
